src/swa.d.ts

```diff
@@ -183,6 +186,11 @@ declare type SWACLISharedLoginOptions = LoginDetails & {
 
 declare type SWACLILoginOptions = SWACLISharedLoginOptions;
 
+declare type SWACLITelemetryOptions = {
+  disable?: boolean;
+  enable?: boolean;
+  status?: boolean;
```

Comment:
Q: what is the experience in case we don't specify all these options will we get the capture telemetry status ?

Reply:
if we dont specify one of the flags, just `swa telemetry`, then nothing happens, wont show the status.